src/codeu/chat/common/BasicController.java

```diff
@@ -49,6 +49,6 @@
   //  operation is successful, a Conversation object will be returned
   //  representing the full state of the conversation on the server.
   //  Whether conversations can have the same title is undefined.
-  ConversationHeader newConversation(String title, Uuid owner);
+  ConversationHeader newConversation(String title, Uuid owner, Integer defaultByte);
```

Comment:
Also (here and elsewhere) defaultByte isn't a very informative name. defaultPermission (that's what this is, right?) would be much clearer.